docs/cookbook/sending_notifications.md

```diff
@@ -0,0 +1,99 @@
+# Sending notifications
+
+You can send two types on notifications to the users.
+
+## Display notification bars
+
+You can have your PHP code send notification that will be displayed as a message bar in the Back Office.
+To do that, inject the `NotificationHandlerInterface` into your class.
+You can use one of the four methods representing four notification types: `info`, `success`, `warning` and `error`.
+
+``` php
+$this->notificationHandler->info('Notification text');
+```
+
+To have the notification translated, make use of the `TranslatorInterface`.
+You need to provide the message strings in the translation files under the correct domain and key.
+
+``` php
+$this->notificationHandler->info(
+    $this->translator->trans(
+        'example.notification.text',
+        [],
+        'domain'
+    )
+);
+```
+
+![Example of an info notification](img/notification2.png)
+
+## Create custom notifications using the Flex Workflow mechanism
+
+!!! enterprise
+
+    You can send your own custom notifications to the user
+    with the same mechanism that is used to send notification from Flex Workflow.
```

Comment:
How does the whole chapter differ from the one above ?

Reply:
You're right, I've added a clarification with screenshots.